channel2/options.go

```diff
@@ -16,17 +16,45 @@
 
 package channel2
 
-import "encoding/json"
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+)
+
+const (
+	defaultOutstandingConnects = 16
```

Comment:
probably should update the release notes now while these settings are fresh in your head?